pkg/ebpf/httpfltr/httpfltr.go

```diff
@@ -227,3 +242,42 @@ func (p *Tracer) serviceName(pid uint32) string {
 	activePids.Add(pid, name)
 	return name
 }
+
+func findNamespace(pid int32) (uint32, error) {
```

Comment:
I'm not sure if this works at all on non Linux, maybe this code needs to be moved to Linux specific file and we should return 0, err on other systems. Returning 0 means we don't support namespaced PIDs.